Security (Is it safe?)/Data security/Vulnerability disclosure program.yaml

```diff
@@ -9,6 +9,13 @@ criterias:
       - indicator: >+
           The company has a mechanism (ex: a bug bounty program) through which security researchers can
           submit vulnerabilities they discover.
+          
+          A company with a bug bounty program has an alternative mechanism to report vulnerabilities that non-technical 
+          consumers can use to report vulnerabilities.
+          
+          The company provides a clear description with examples of what is defined as a security vulnerability.
+          
+          The company provides a security@company email address or submission form that manages vulnerabilities only.
```

Comment:
Should we give companies some room for naming their delicate email addresses which is for managing vulnerabilities only? How about "The company provides a delicate email address or submission form that manages vulnerabilities only."